DSCResources/MSFT_xWebsite/MSFT_xWebsite.schema.mof

```diff
@@ -10,17 +10,17 @@ Class MSFT_xWebBindingInformation
 };
 
 
-[ClassVersion("2.0.0"), FriendlyName("xWebsite")] 
+[ClassVersion("2.0.0"), FriendlyName("xWebsite")]
 class MSFT_xWebsite : OMI_BaseResource
 {
   [Key] string Name;
-  [write] string PhysicalPath;
+  [Required] string PhysicalPath;
```

Comment:
After reviewing, I think we should optimize for the 80% case not the 20%.  We should require the PhysicalPath.